packages/core/admin/admin/src/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Dummy tsconfig if we want to do TS in the codebase

Reply:
Why don't you just extend the one from @strapi/typescript-utils?

Reply:
+ modify the include/exclude